tests/shell/tests.py

```diff
@@ -287,8 +320,12 @@ def get_auto_imports(self):
     def test_message_with_stdout_no_installed_apps(self):
```

Comment:
We have lost coverage of "0 objects" being in the console, I would write a test with `get_auto_imports` overridden to return an empty list.